packages/venia-ui/lib/components/Image/simpleImage.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { string, func } from 'prop-types';
+
+const SimpleImage = props => {
+    const { className, handleError, handleLoad, src, ...rest } = props;
+
+    return (
+        // eslint-disable-next-line jsx-a11y/alt-text
```

Comment:
This lint rule exists for a good reason. Let's make sure `alt` is a required prop. 😅

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/1871/commits/1c31db4905410ce8f09baa55066c636086f2cd85.